packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1732,6 +1733,11 @@ class Scaffold extends StatefulWidget {
   /// Optional callback that is called when the [Scaffold.endDrawer] is opened or closed.
   final DrawerCallback? onEndDrawerChanged;
 
+  /// Optional callback that is called when the Scaffold's status bar is tapped.
+  ///
+  /// This callback is only called in iOS and macOS applications.
```

Comment:
The status bar gesture detector is currently only added for iOS and macOS platforms in the material scaffold. We could consider expanding across platforms.

Reply:
Ah yes, I recall that is to match the iOS scroll to top feature, which isn't present in other platforms. If we were to expose it to everyone, it would be strange to limit it to a given platform.